src/styles/global.css

```diff
@@ -88,6 +88,9 @@
 .text-shadow {
   text-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
 }
+.text-stoke {
+  -webkit-text-stroke: 1px #333;
```

Comment:
Try `1.5px var(--color-light-cream)`. I think it looks way better to match our design.

Also, consider adding it as the default option for all h1 - h3 headings!